src/lib/undux/utils/withdraw.js

```diff
@@ -49,7 +49,8 @@ export const executeWithdraw = async (store: Store, code: string, reason: string
             res({ status, transactionHash })
           },
           onError: e => {
-            userStorage.markWithErrorEvent(e)
+            const receipt = JSON.parse(`{${e.message.split('{')[1]}`)
+            userStorage.handleReceiptUpdated(receipt)
```

Comment:
1. i didnt want this. just wondering if we can txhash in a simpler way.
restore the call to simply update the status of the event.
2. some errors do not return receipt. so we need to handle this case. maybe also save txhash from ontxhash

Reply:
1. Done. reverted the changes.
2. The transaction hash is already being saved in onTxHash callback.

Reply:
i ment use txhash from ontxhash in error handling.
save txhash to outerscope

Reply:
Done